jupyter-js-widgets/css/widgets-base.css

```diff
@@ -350,11 +361,6 @@
 /* Widget Slider */
 
 .widget-slider {
-    /* Local Variables for the slider */
-    --jp-widgets-track-thickness: 6px;
-    --jp-widgets-handle-thickness: 12px;
-    --jp-widgets-handle-size: var(--jp-widgets-inline-height);
-    --jp-widgets-slider-border-width: var(--jp-widgets-border-width);
```

Comment:
See the merge commit. Basically, the preprocessors need them in root because the preprocessors don't know about the DOM structure, so they can't correctly scope them.